test/validate/ferc1_test.py

```diff
@@ -5,49 +5,13 @@
 """
 
 import logging
-from itertools import chain
 
-import pandas as pd
 import pytest
 
 from pudl import validate as pv
-from pudl.etl import defs
-from pudl.extract.ferc1 import TABLE_NAME_MAP_FERC1
-from pudl.metadata.classes import DataSource
-from pudl.transform.ferc import filter_for_freshest_data_xbrl, get_primary_key_raw_xbrl
 
 logger = logging.getLogger(__name__)
 
-# These are tables for which individual records have been sliced up and
-# turned into columns (except for the transmission table) -- so there's no universally
-# unique record ID. But we should parameterize the has_unique_record_ids class
-# attributes in the FERC classes.
-non_unique_record_id_tables = [
-    "core_ferc1__yearly_plant_in_service_sched204",
-    "core_ferc1__yearly_purchased_power_and_exchanges_sched326",
-    "core_ferc1__yearly_energy_sources_sched401",
-    "core_ferc1__yearly_energy_dispositions_sched401",
-    "core_ferc1__yearly_utility_plant_summary_sched200",
-    "core_ferc1__yearly_transmission_lines_sched422",
-    "core_ferc1__yearly_balance_sheet_liabilities_sched110",
-    "core_ferc1__yearly_balance_sheet_assets_sched110",
-    "core_ferc1__yearly_income_statements_sched114",
-    "core_ferc1__yearly_depreciation_summary_sched336",
-    "core_ferc1__yearly_depreciation_changes_sched219",
-    "core_ferc1__yearly_depreciation_by_function_sched219",
-    "core_ferc1__yearly_operating_expenses_sched320",
-    "core_ferc1__yearly_cash_flows_sched120",
-    "core_ferc1__yearly_retained_earnings_sched118",
-    "core_ferc1__yearly_operating_revenues_sched300",
-    "core_ferc1__yearly_other_regulatory_liabilities_sched278",
-    "core_ferc1__yearly_sales_by_rate_schedules_sched304",
-]
-unique_record_tables = [
-    t
-    for t in DataSource.from_id("ferc1").get_resource_ids()
-    if t not in non_unique_record_id_tables
-]
```

Comment:
No longer required, as these checks for uniqueness have been implemented in `dbt`